src/autopas/selectors/AutoTuner.h

```diff
@@ -457,74 +459,87 @@ bool AutoTuner<Particle>::iteratePairwise(PairwiseFunctor *f, bool doListRebuild
  * @tparam PairwiseFunctor
  * @param f
  * @param containerPtr
- * @param particleBuffer
- * @param haloParticleBuffer
+ * @param particleBuffers
+ * @param haloParticleBuffers
  * @note If the buffers are replaced with actual cells, we could use the CellFunctor to simplify things and potentially
  * even use SoA.
  */
 template <bool newton3, class Particle, class T, class PairwiseFunctor>
-void doRemainderTraversal(PairwiseFunctor *f, T containerPtr, std::vector<Particle> &particleBuffer,
-                          std::vector<Particle> &haloParticleBuffer) {
-  // 1. particleBuffer with all close particles in container
-  for (auto &&p : particleBuffer) {
-    auto pos = p.getR();
-    auto min = autopas::utils::ArrayMath::subScalar(pos, containerPtr->getCutoff());
-    auto max = autopas::utils::ArrayMath::addScalar(pos, containerPtr->getCutoff());
-    for (auto iter2 = containerPtr->getRegionIterator(min, max, IteratorBehavior::ownedOrHalo); iter2.isValid();
-         ++iter2) {
-      if (newton3) {
-        f->AoSFunctor(p, *iter2, true);
-      } else {
-        f->AoSFunctor(p, *iter2, false);
-        f->AoSFunctor(*iter2, p, false);
+void doRemainderTraversal(PairwiseFunctor *f, T containerPtr, std::vector<std::vector<Particle>> &particleBuffers,
```

Comment:
doRemainderTraversal should have an expanded description, this is too vague.

Reply:
done [3709cf3](https://github.com/AutoPas/AutoPas/pull/702/commits/3709cf3bf23b442b7cd06514a1cc10f036b620a6)